File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/VoidTicketResponseContent.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

import de.dimedis.mobileentry.model.Action;
import de.dimedis.mobileentry.model.Ticket;

public class VoidTicketResponseContent extends BaseResponseContent {
    @SerializedName("ticket_code")
    public String ticketCode;

    @SerializedName("ticket")
    public Ticket ticket;

    @SerializedName("action")
    public Action action;
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/VoidTicketResponseContent.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName
import de.dimedis.mobileentry.model.Action
import de.dimedis.mobileentry.model.Ticket

class VoidTicketResponseContent : BaseResponseContent() {
    @SerializedName("ticket_code")
    var ticketCode: String? = null

    @SerializedName("ticket")
    var ticket: Ticket? = null

    @SerializedName("action")
    var action: Action? = null
}